modules/enabling-data-science-pipelines-2.adoc

```diff
@@ -109,11 +161,11 @@ Before removing the OpenShift Pipelines Operator, ensure that migration of your
 [role="_additional-resources"]
 .Additional resources
 
+* link:https://pypi.org/project/kfp/[PyPI:]
```

Comment:
Not a must, but suggest including caret symbols so that web browsers open these external links in new tabs. Example `link:https://pypi.org/project/kfp/[PyPI:^]`

Reply:
Good idea, updated.